### ISSUE TITLE
Cannot use *_material anymore

### ISSUE THREAD

**Issue by tiramiseb** [terraform-providers/terraform-provider-docker#86]
Hello,

I have a big problem with initializing the provider itself.

### Terraform & provider versions

Terraform v0.11.7
Docker provider v1.0.1

### Affected Resource

The provider itself.

### Terraform Configuration Files
```hcl
provider "docker" {
  host          = "tcp://<some IP>:<some port>"
  ca_material   = "${data.blahblah_ca}"
  cert_material = "${data.blahblah_cert}"
  key_material  = "${data.blahblah_key}"
}
```

### Expected Behavior
Use the content of `data.blahblah_*` as CA, cert and key material.

### Actual Behavior
Use the content of `data.blahblah_*` as CA, cert and key files paths.

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
It certainly comes from this commit:
https://github.com/terraform-providers/terraform-provider-docker/commit/1f563cc913df3a153f9aef0dd3171c65108f8819#diff-4a9f60e1d3ebb2215df43a2c3d83cfa8
... where:
```
dc.NewTLSClientFromBytes(c.Host, []byte(c.Cert), []byte(c.Key), []byte(c.Ca))
```
... has been replaced with:
```
client.NewClientWithOpts(
	client.WithHost(c.Host),
	client.WithTLSClientConfig(c.Ca, c.Cert, c.Key),
	client.WithVersion(apiVersion),
)
```

`client.WithTLSClientConfig(c.Ca, c.Cert, c.Key)` (from github.com/docker/docker/client) takes files paths as arguments, not data contents.

**Comment by mavogel** [comment]
Hi @tiramiseb 
yes you're right! This breaking change was unintentionally introduced while changing the underlying library in #32. With the gained knowledge, I guess the right approach is to accept only data contents and if the user decided to use file paths, he has to make use of the interpolation `file` function (https://www.terraform.io/docs/configuration/interpolation.html#file-path-)

Let me fix it in the next minor release. Until then if you do not need the latest swarm/service features fix the provider version.
Manu

Related to #75

**Comment by mavogel** [comment]
Will be fixed with #91 @tiramiseb in `v1.0.2`

**Comment by tiramiseb** [comment]
Thanks a lot :)
